pkg/controller/installation/core_controller.go

```diff
@@ -1309,9 +1336,22 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 		ManagerInternalSecret:       managerInternalTLSSecret,
 		Terminating:                 terminating,
 		UsePSP:                      r.usePSP,
+		MetricsServerTLS:            kubeControllerTLS,
+		TrustedBundle:               kubeControllerTrustedBundle,
 	}
 	components = append(components, kubecontrollers.NewCalicoKubeControllers(&kubeControllersCfg))
 
+	components = append(components,
```

Comment:
The trusted bundles are a per-namespace thing as it currently is implemented. With this addition two configmaps are competing for one spot. This block has to be removed. The certificates added in this PR, should be in the `typhaNodeTLS.TrustedBundle`

Reply:
Done